CourseApp.Tests/AddTests.cs

```diff
@@ -0,0 +1,106 @@
+﻿namespace CourseApp.Tests
+{
+    using Xunit;
+
+    public class AddTests
+    {
+        [Fact]
+        public void AddIntsNegNegNeg()
+        {
+            int a = -3;
+            int b = -4;
+            int expected = -7;
+
+            int actual = new Calculator().Add(a, b);
+            Assert.Equal(actual, expected);
```

Comment:
Please, write first expected value and then actual
According to xUnit docs: `Assert.Equal(expected, actual)`

Reply:
Fixed